.gitlab/macrobenchmarks.yml

```diff
@@ -0,0 +1,221 @@
+variables:
+  BENCHMARKS_CI_IMAGE: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/benchmarking-platform:go-go-prof-app
+
+.benchmarks:
+  stage: macrobenchmarks
+  tags: ["runner:apm-k8s-same-cpu"]
+  timeout: 1h
+  rules:
+    - if: $CI_PIPELINE_SOURCE == "schedule"
+      when: always
+    - when: manual
+  # If you have a problem with Gitlab cache, see Troubleshooting section in Benchmarking Platform docs
+  image: $BENCHMARKS_CI_IMAGE
+  script:
+    - git clone --branch go/go-prof-app https://gitlab-ci-token:${CI_JOB_TOKEN}@gitlab.ddbuild.io/DataDog/benchmarking-platform platform && cd platform
+    - "./generate-run-plan-and-run-benchmarks.sh"
+  artifacts:
+    name: "artifacts"
+    when: always
+    paths:
+      - artifacts/
+  variables:
+    FF_USE_LEGACY_KUBERNETES_EXECUTION_STRATEGY: "true" # Important tweak for stability of benchmarks
+    KUBERNETES_SERVICE_ACCOUNT_OVERWRITE: dd-trace-go
+
+  # Workaround: Currently we're not running the benchmarks on every PR, but GitHub still shows them as pending.
+  # By marking the benchmarks as allow_failure, this should go away. (This workaround should be removed once the
+  # benchmarks get changed to run on every PR)
+  allow_failure: true
+
+go118-baseline:
+  extends: .benchmarks
+  variables:
+    ENABLE_DDPROF: "false"
+    ENABLE_TRACING: "false"
+    ENABLE_PROFILING: "false"
+    ENABLE_APPSEC: "false"
+    DD_INSTRUMENTATION_TELEMETRY_ENABLED: "true"
+    DD_INSTRUMENTATION_TELEMETRY_DEBUG: "true"
+    DD_PROFILING_EXECUTION_TRACE_ENABLED: "false"
+    GO_VERSION: "1.18"
+    GO_PROF_APP_BUILD_VARIANT: candidate
+    DD_TRACE_GO_VERSION: latest
+    LOAD_TESTS: normal_operation_io-bound,high_load_io-bound|normal_operation_cpu-bound,high_load_cpu-bound|normal_operation_cgo-cpu-bound,high_load_cgo-cpu-bound|normal_operation_cpu-bound-x-client-ip-enabled,high_load_cpu-bound-x-client-ip-enabled
+    PARALLELIZE: "true"
```

Comment:
Can you please move these common variables to base `.benchmarks` job, so we don't need to copy them over?